### PR TITLE
Disable status page. r=dustin,eli

### PR DESCRIPTION
**Pull request by walac**
It never got really production ready and it is blocking us from
disabling cors-proxy.

### CODE REVIEW

File: .neutrinorc.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const envs = {
   UPTIMEROBOT_API_KEY_INDEX: 'm776362434-85a6996de0f9c73cf21bbf89',
   UPTIMEROBOT_API_KEY_SCHEDULER: 'm776120202-44923d8660c2a1bd1a5de440',
   UPTIMEROBOT_API_KEY_SECRETS: 'm777577313-6d58b81186c4064cf7a8d1e1',
-  SIGN_IN_METHODS: process.env.NODE_ENV === 'development' ? 'development' : 'okta email manual'
```
Comment (djmitche) [inline .neutrinorc.js]: not part of this change?

**Comment by walac** [comment]
> And rm src/views/Status?

I am leaving the code for posterity for now :)

**Comment by eliperelman** [comment]
Why? We have git history for that.

**Comment by walac** [comment]
> Why? We have git history for that.

https://en.wikipedia.org/wiki/Principle_of_least_effort :)

**Comment by eliperelman** [comment]
LOL, yeah let's remove it.

**Comment by djmitche** [comment]
https://github.com/djmitche/taskcluster-tools/commit/2253ca164a01411ed90cf8767d4d49ac28bba39d

For some reason I can't push that to your branch though..

**Comment by helfi92** [comment]
He would probably have to give you permission to push to his fork.

**Comment by djmitche** [comment]
Maybe, but I think that box is checked (Github says "Add more commits by pushing to the master branch on walac/taskcluster-tools.") and the error is "non fast-forward" even though it's a fast-forward push..

**Comment by walac** [comment]
Sorry, it was having local build issues, removed the Status code.
